model/clusters_mgmt/v1/add_on_inqueries_resource.model

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages add-on inqueries, inqueries perform validation of add-on(s) on a per cluster basis
+// based on add-on conditions and requirements.
+resource AddonInqueries {
+    method List {
+        // Index of the requested page, where one corresponds to the first page.
+        in out Page Integer = 1
+
+        // Maximum number of items that will be contained in the returned page.
+        in out Size Integer = 100
+
+        // Search criteria.
+        //
+        // The syntax of this parameter is similar to the syntax of the _where_ clause of an
+        // SQL statement, but using the names of the attributes of the add-on instead of
+        // the names of the columns of a table. For example, in order to retrieve all the
+        // add-ons with a name starting with `my` the value should be:
+        //
+        // [source,sql]
+        // ----
+        // name like 'my%'
+        // ----
+        //
+        // If the parameter isn't provided, or if the value is empty, then all the add-ons
+        // that the user has permission to see will be returned.
+        in Search String
+
+        // Order criteria.
+        //
+        // The syntax of this parameter is similar to the syntax of the _order by_ clause of
+        // a SQL statement, but using the names of the attributes of the add-on instead of
+        // the names of the columns of a table. For example, in order to sort the add-ons
+        // descending by name the value should be:
+        //
+        // [source,sql]
+        // ----
+        // name desc
+        // ----
+        //
+        // If the parameter isn't provided, or if the value is empty, then the order of the
+        // results is undefined.
+        in Order String
+
+        // Total number of items of the collection that match the search criteria,
+        // regardless of the size of the page.
+        out Total Integer
+
+        // Retrieved list of add-ons.
+        out Items []AddOn
+    }
+
+    locator AddonInquery {
+        target AddonInquery
+        variable AddOnID
```

Comment:
Can we make it consistent lowercase 'AddonID' or is there some limitation?

Reply:
done :+1: